content/en/continuous_integration/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Continuous Integration Visibility
+kind: documentation
+further_reading:
+    - link: "/continuous_integration/explore_pipelines/"
+      tag: "Documentation"
+      text: "Start exploring pipeline data to resolve build problems"
+    - link: "/continuous_integration/explore_tests/"
+      tag: "Documentation"
+      text: "Start exploring tests data to find and fix problem tests"
+---
+
+<div class="alert alert-info"><p>CI Visibility is in private beta. There are no billing implications for tracing pipelines and tests during this period. If you want to be added to the private beta, <a href="http://dtdg.co/ci-visibility-beta-request">fill out this form</a>.</p><p>CI Visibility is available only on <a href="/getting_started/site/">the US1 Datadog site</a> at this time.</p>
```

Comment:
The form requires the organization ID of the customer which is not very straightforward to get. Can we send them to https://app.datadoghq.com/ci/getting-started instead which contains a link that pre-populates the form with their name/email/org ID?